### DISCUSSION TITLE
Security Information

### DISCUSSION THREAD

**Discussion by aviv85503** [rustdesk/rustdesk#9835] [Q&A] [answered]
can i have a document or more information about rustdesk security?

**Comment by rustdesk** [comment, marked as answer]
Here is a summary.
- How does RustDesk work: https://github.com/rustdesk/rustdesk/wiki/How-does-RustDesk-work%3F
- Encryption:
  - https://github.com/rustdesk/rustdesk/discussions/9172 
  - https://github.com/rustdesk/rustdesk/discussions/2239#discussioncomment-5647068
- https://github.com/rustdesk/rustdesk/discussions/5624 
- ACL: https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/permissions/
- 2FA: https://github.com/rustdesk/rustdesk/pull/6782
- Password protection: https://github.com/rustdesk/rustdesk/discussions/8197
- Control role: https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/control-role/
- Administration role: https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/admin-role/
